### PR TITLE
Adding forum structure changes blog post

### PR DESCRIPTION
**Pull request by euantorano**
As the structural changes are still a work in progress and may well continue to change over the next couple of days, this is a work in progress blog post to announce the changes.

### CODE REVIEW

File: december-2017/forum-structure-changes.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+- [Planned Features](https://community.mybb.com/forum-194.html) - this forum is used for any features that have been confirmed to be part of a future release. Some of the threads here may be current works in progress, or may even be fully implemented already.
+- [Rejected Features](https://community.mybb.com/forum-195.html) - this forum is used for any features that have been rejected. This could be due to the request being a duplicate of another planned feature, or due to it being outside of the scope of MyBB's current direction.
+
+This restructuring is still currently a work in progress, and as such the old [1.8 Suggestions and Feedback](https://community.mybb.com/forum-158.html) forum still exists, though new threads cannot be created there. The same goes for the [2.0 Suggestions and Feedback forum](https://community.mybb.com/forum-152.html).
+
+As ever, we welcome any feedback on this new structure and are always lookin for ways to improve visibility into the direction that we are heading. We hope that by restructuring in these kinds of way the future of development should become more clear.
```
Comment (Ben-MyBB) [inline december-2017/forum-structure-changes.md]: As ever, we welcome any feedback on this new structure and are always looking for ways to improve visibility into the direction that MyBB is heading. 

Slight spelling mistake.

Comment (Ben-MyBB) [inline december-2017/forum-structure-changes.md]: We hope that by restructuring in these kinds of way the future of development should become more clear.

This doesn't sound quite right to me but I can't seem to word it right either.

**Review by Ben-MyBB — COMMENTED** [review]
Just a few small things

File: december-2017/forum-structure-changes.md (hunk 1 above)
Comment (euantorano) [reply december-2017/forum-structure-changes.md]: Anybody got any thoughts on possible rewording @mybb/community-team

**Comment by euantorano** [comment]
Any more feedback from anyone before we publish? Would like to publish this week if possible.

**Pull request closed without merging**